### ISSUE TITLE
Give me the data.

### ISSUE THREAD

**Issue by ourin0523** [namdvt/HDR-reconstruction-using-deep-CNNs#1]
Nice to meet you.
I would like to use your program, but could you also upload the "data/train" that is referenced in "split_image.py"?

**Comment by namdvt** [comment]
As I mentioned in README. the data is downloaded from this link https://cseweb.ucsd.edu/~viscomp/projects/SIG17HDR/

**Comment by ourin0523** [comment]
I'm sorry.
I don't know because I'm an idiot, so please tell me.
I downloaded both "Test set" and "Training set" from your site, but I don't know how to create the "data/train" used in "split_image.py", so I wanted to ask you.
Please let me know.

**Comment by namdvt** [comment]
After downloaded the "training" data and extract, you will see the ldr and hdr are placed in subfolders. You should modify the split_image.py to your dataset folder.
 
In addition, in this project I only use the median ldr image and target hdr for the training. So you need to change the source code to split these images and place them in to approriate folders.

If you are not familiar with python, I suggest you to finish some python tutorial/course before diving into deep learning.
